src/index.ts

```diff
@@ -21,6 +21,14 @@ interface GenericObject {
   [key: string]: string;
 }
 
+export interface CustomDNS {
```

Comment:
suggestion to rename to CustomDns

Reply:
> Why did you pen another MR? 👀

I didn't know I had to use commitlint to commit that's why I redid it using commitlint